.evg.yml

```diff
@@ -108,6 +117,21 @@ functions:
           ./resources/run_adl.sh stop
           exit $EXITCODE
 
+  "run smoke test":
+    command: shell.exec
+    type: test
+    params:
+      working_dir: mongo-jdbc-driver
+      script: |
+        ${PREPARE_SHELL}
+        ./resources/run_adl.sh start &&
```

Comment:
@nbagnard We discussed about adding the start/stop adl as a pre/post functions, I found that the pre/post steps run for all tasks.  Also that tasks are independent of each other, such as being started in different directories.  Starting it in the task is the way to run the local adl.

Reply:
I see. Yes, it makes sense.